core/presentation/designsystem/src/main/java/com/nolawiworkineh/designsystem/components/TaskyBackground.kt

```diff
@@ -0,0 +1,114 @@
+package com.nolawiworkineh.designsystem.components
+
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun TaskyBackground(
+    blackHeightFraction: Float = 0.16f, // Default to 16% height for the black container
+    content: @Composable () -> Unit, // Slot for dynamic content
```

Comment:
Tip, if you move the `content` parameter to the last position, you can write the following, which is much more idiomatic and practical:

```
TaskyBackground(
  toolbar = { Text(text = "Title") },
  isBlackToolBar = false
) {
  // Here you can add your Compose code for your `content` parameter
}
```

This is because of Kotlin´s trailing lambda feature. You´ve been using the same structure with:

```
Row { }
Column { }
Box { }
```